code/game/gamemodes/revolution/revolution.dm

```diff
@@ -173,18 +173,35 @@
 /datum/game_mode/revolution/set_round_result()
 	..()
 	if(finished == 1)
-		SSticker.mode_result = "win - heads killed"
-		SSticker.news_report = REVS_WIN
+		if(crew_live_count(50))
+			SSticker.mode_result = "win - heads killed"
+      SSticker.news_report = REVS_WIN
+		else
+			SSticker.mode_result = "halfwin - heads killed - too many died"
+      SSticker.news_report = REVS_WIN_COSTLY
 	else if(finished == 2)
-		SSticker.mode_result = "loss - rev heads killed"
-		SSticker.news_report = REVS_LOSE
+		if(crew_live_count(50))
+			SSticker.mode_result = "loss - revheads killed"
+			SSticker.news_report = REVS_LOSE
+		else
+			SSticker.mode_result = "halfwin - revheads killed - too many died"
+      SSticker.news_report = REVS_LOSE_COSTLY
+	..()
+	return TRUE
 
 //TODO What should be displayed for revs in non-rev rounds
 /datum/game_mode/revolution/special_report()
 	if(finished == 1)
-		return "<span class='redtext big'>The heads of staff were killed or exiled! The revolutionaries win!</span>"
+    if(crew_live_count(50))
+		  return "<FONT size = 3><B>Revolution Major Success</B></FONT></br><span class='redtext'>The heads of staff were killed or exiled! The revolution was a great success!</span>"
```

Comment:
Please use the spans and appropriate classes

Reply:
I stole the format from nukeops unless you recently changed them to give them actual spans.